frontend/src/store/actions.js

```diff
@@ -14,6 +16,20 @@ export const actions = {
       commit(mutations.SET_USER, user);
     });
   },
+  [FETCH_CARS]({ commit }) {
+    commit(mutations.SET_FETCHING_CARS_PROGRESS, true);
```

Comment:
probably should set errors to null here too

Reply:
An error is set to null after getCars was resolved successfully. If I understand you correctly I should move this earlier, before fetching data?